src/fix/main.c

```diff
@@ -653,6 +653,24 @@ static ssize_t writeBytes(int fd, void *buf, size_t len)
 	return total;
 }
 
+/**
+ * @param rom0 A pointer to rom0
+ * @param startAddr What address to begin checking from
+ * @param size How many bytes to check
+ * @param areaName Name to be displayed in the warning message
+ */
+static void warnNonZero(uint8_t *rom0, uint16_t startAddr, uint8_t size, char const *areaName) 
+{
+	for (uint8_t i = 0; i < size; i++) {
+		if (rom0[i + startAddr] != 0) {
+			fprintf(stderr, "warning: Overwrote %s in the %s\n",
+				size > 1 ? "one or more non-zero bytes" : "the non-zero byte",
```

Comment:
```suggestion
				"a non-zero byte",
```
Though at this point it should be merged into the format string anyway.